internal/topo/topo.go

```diff
@@ -120,8 +125,37 @@ func (s *Topo) addEdge(from api.TopNode, to api.TopNode, toType string) {
 // stream starts execution.
 func (s *Topo) prepareContext() {
 	if s.ctx == nil || s.ctx.Err() != nil {
-		contextLogger := conf.Log.WithField("rule", s.name)
-		ctx := kctx.WithValue(kctx.Background(), kctx.LoggerKey, contextLogger)
+		ctxLogger := &logrus.Logger{
```

Comment:
Are there cases to reuse logger as before?

Reply:
If we reused logger, any changes for rule logger will affect the global logger, which I believe is not what we expect.

Reply:
That is the default behavior previously so it should be harmless. With this change, will it have problems when writing to the same file simultaneously by multiple rules?

Reply:
Fixed.